agate/table/from_csv.py

```diff
@@ -75,9 +79,9 @@ def from_csv(cls, path, column_names=None, column_types=None, row_names=None, sk
                 next(reader)
 
         if row_limit is None:
-            rows = tuple(reader)
+            rows = reader
         else:
-            rows = tuple(itertools.islice(reader, row_limit))
+            rows = itertools.islice(reader, row_limit)
```

Comment:
I don't think this saves any time, and it causes an error in `Table.__init__()`.

Reply:
Now fixed by allowing an iterator in `Table.__init__`. Avoiding the `tuple` here allows the file to stay on disk rather than pulling it all into memory.